pkg/sidecar/fanout/fanout_handler.go

```diff
@@ -40,6 +40,7 @@ const (
 // Configuration for a fanout.Handler.
 type Config struct {
 	Subscriptions []eventingduck.ChannelSubscriberSpec `json:"subscriptions"`
+	AsyncHandler  bool                                 `json:"asyncHandler"`
```

Comment:
My preference is `Async`, rather than `AsyncHandler`.

It should probably omit on empty as well.

```golang
Async bool `json:"async,omitempty"`
```

Reply:
I can change but just Async sounds a bit too ambiguous, where AsyncHandler clearly states what that is. Please let me know if you still want me to change it.

Reply:
I have a minor preference for `async`, but I can see why you prefer `asyncHandler`. Unless someone else chimes in, leave it as-is.